Saucier720-app/src/app/deals/FEC/deals-store-button/deals-store-button.component.ts

```diff
@@ -13,9 +13,17 @@ import { Store } from 'src/app/core/interfaces/store';
 })
 
 export class DealsStoreButtonComponent {
-  name: string = '';
+  name: string = 'Walmart';
 
-  constructor(private dealsService: DealsService) { }
+  constructor(private dealsService: DealsService) {
+    const buttonStateJson = localStorage.getItem('buttonState');
```

Comment:
why is this var name "buttonStateJson"?

Reply:
@rileycleavenger its checking for a Json item and if it exists it can parse it. Maybe it can just be jsonItem or something

Reply:
ah I see, I thought it might have been leftover from my last button state thing